src/selectors/team.js

```diff
@@ -2,6 +2,7 @@ import { createSelector } from 'reselect';
 
 export const getTeam = state => state.team;
 export const getTeamDefaultZoom = state => state.team.default_zoom;
+export const getTeamVoteDuration = state => state.team.vote_duration;
```

Comment:
I'm going to remove this as part of the revision, as it doesn't get used anywhere.